network/errors.go

```diff
@@ -3,6 +3,9 @@ package network
 import "errors"
 
 var (
-	errSubnetV6NotFound = errors.New("Couldn't find ipv6 subnet in network info")
-	errV6SnatRuleNotSet = errors.New("ipv6 snat rule not set. Might be VM ipv6 address missing")
+	errSubnetV6NotFound        = errors.New("Couldn't find ipv6 subnet in network info")                // nolint
+	errV6SnatRuleNotSet        = errors.New("ipv6 snat rule not set. Might be VM ipv6 address missing") // nolint
+	ErrEndpointStateNotFound   = errors.New("endpoint state could not be found in the statefile")
+	ErrConnectionFailure       = errors.New("couldn't connect to CNS daemonset")
```

Comment:
probably not daemonset?
```suggestion
	ErrConnectionFailure       = errors.New("couldn't connect to CNS")
```